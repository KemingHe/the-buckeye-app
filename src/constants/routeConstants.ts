export const HOME_ROUTE: string = "/";

// Authentication routes.
export const AUTH_SIGNIN_ROUTE: string = "/auth/signin";
export const AUTH_VERIFY_ROUTE: string = "/auth/verify";
export const AUTH_SIGNOUT_ROUTE: string = "/auth/signout";

// User routes.
export const USER_DASHBOARD_ROUTE: string = "/dashboard";

// Events route.
export const EVENTS_ROUTE: string = "/events";

// Help and support route.
export const SUPPORT_ROUTE: string = "/support";
