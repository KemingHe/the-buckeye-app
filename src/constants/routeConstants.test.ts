import { describe, expect, it } from "vitest";

import * as routes from "./routeConstants";

describe("routeConstants", () => {
  it("matches the snapshot", () => {
    expect(routes).toMatchSnapshot();
  });
});
