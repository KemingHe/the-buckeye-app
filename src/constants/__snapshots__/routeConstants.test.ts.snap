// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`routeConstants > matches the snapshot 1`] = `
{
  "AUTH_SIGNIN_ROUTE": "/auth/signin",
  "AUTH_SIGNOUT_ROUTE": "/auth/signout",
  "AUTH_VERIFY_ROUTE": "/auth/verify",
  "EVENTS_ROUTE": "/events",
  "HOME_ROUTE": "/",
  "SUPPORT_ROUTE": "/support",
  "USER_DASHBOARD_ROUTE": "/dashboard",
}
`;
